arangodb/license.md

```diff
@@ -1 +1,7 @@
-[ArangoDB itself is licensed under the Apache License](https://github.com/arangodb/arangodb/blob/devel/LICENSE), but it contains [software of third parties under their respective licenses](https://github.com/arangodb/arangodb/blob/devel/LICENSES-OTHER-COMPONENTS.md).
+The official Docker images of the ArangoDB Community Edition are governed by the [ArangoDB Community License](https://arangodb.com/2024/02/update-evolving-arangodbs-licensing-model-for-a-sustainable-future/). The use for commercial purposes is limited to a 100 GB on dataset size in production within a single cluster and a maximum of three clusters.
```

Comment:
Is the text for the ArangoDB Community License available anywhere? I could not find a link to it in the blog post.

Reply:
No, the blog post is the license text at the moment. I asked this internally, too, but there is no other source it seems.